### DISCUSSION TITLE
is it a rlink1 or 2? they done 2 versions.

### DISCUSSION THREAD

**Discussion by Nikoletapex** [cedricp/ddt4all#881] [Q&A]
is it a rlink1 or 2? they done 2 versions.

This is rlink1 in the picture , does yours look like this?:
![r-link evolution](https://github.com/cedricp/ddt4all/assets/49172217/f91f6f4c-2ee1-47cd-8bd6-c9fe482d7986)

_Originally posted by @AndyIreland in https://github.com/cedricp/ddt4all/issues/877#issuecomment-1656707329_
         When the screen opens it sais rlink, plain  and its not coloufoul.the software system is 2.2.19.300
![IMG_20230728_171608](https://user-images.githubusercontent.com/140859711/256981188-11f79aae-122e-4931-b717-1e0e34a82119.jpg)

**Comment by AndyIreland** [comment]
@Nikoletapex  - it looks like you have a rlink2 version there

**Comment by Nikoletapex** [comment]
But when i open the screen it sais r link..anyway what should i do now to recover the computer back to normal?i think that the Renault dealer here has no idea of how to handle it..so your precious help is my only hope!
I forgort to mention one more problem..there was the potential to change the colour of the cadran(the plasce where it shows the speed of the car),blue,green,red and now it shows the menu complete but doesnt change the colour when you press the button..

**Comment by AndyIreland** [comment]
> But when i open the screen it sais r link..anyway what should i do now to recover the computer back to normal?i think that the Renault dealer here has no idea of how to handle it..so your precious help is my only hope! I forgort to mention one more problem..there was the potential to change the colour of the cadran(the plasce where it shows the speed of the car),blue,green,red and now it shows the menu complete but doesnt change the colour when you press the button..

ah right - sorry i have rlink1so I know a bit about that. I am sure someone else on here will have idea like @KarelSvo and some others they are good on rlink 2 - IO think it has to be relaoded with mid and mid2 etc if there was an error but I am not well knowleged on rlink2 sorry, just some knowledge on rlink1 that is fitted to renault

**Comment by matzel82** [comment]
It's RLink2 !! You can't simple recover...RLink it the hell...

I work at a Renault Garage 🤣🙈

**Comment by AndyIreland** [comment]
@Nikoletapex  if that was 'REALLY' updated , your garage would have updated it to version 3.3 so you could get Android Auto on it (but then going from 2.2 to 3.3 can cause black screen if not done properly or not taken to a garage to have the update done as far as I understand it?) 

if you had an ELM327 OBD2 diagnostic adaptor and specially wired OBD2 cable to read the rlink through the OBD2 socket (you need to have specially wired cable for rlink1, I am not sure whether you need it for rlink2 too?) you (or I could remotely) could just see if outside temp and other parameters are enabled after they did what they did and if not may be able to be enabled again using DDT4ALL or DDT4ALL mode through PYREN

If they were using PYREN software (which I wouldnt have thought so, I would have thought they used CAN CLIP software) but if they did use PYREN , if they done it properly they would have (Should have) backed up/made a dump file of your rlink2 and its settings so that if anything went wrong they could load the dump file back on and get it to how it was.
